pubspec.yaml

```
@@ -71,7 +71,7 @@ flutter:
 
   # To add assets to your application, add an assets section, like this:
   assets:
-    - assets/Arrow-Right.png
+     - assets/images/
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)